src/at_commands/at_ast_definition.rs

```diff
@@ -27,7 +27,7 @@ pub async fn results2message(result: &AstQuerySearchResult) -> Vec<ContextFile>
             file_content: content,
             line1: res.symbol_declaration.full_range.start_point.row + 1,
             line2: res.symbol_declaration.full_range.end_point.row + 1,
-            symbol: res.symbol_declaration.guid.clone(),
+            symbol: vec![res.symbol_declaration.guid.clone()],
```

Comment:
it's not obvious that symbol is a vector now
add a todo in it's definition that we should rename it asap

Reply:
yes, it's already been highlighted by @JegernOUTT

Reply:
done 👌🏻